xbmc/windowing/wayland/GLContextEGL.cpp

```diff
@@ -35,7 +35,7 @@ CGLContextEGL::CGLContextEGL()
   {
     throw std::runtime_error("EGL implementation does not support EGL_EXT_platform_base, cannot continue");
   }
-  if (m_clientExtensions.find("EGL_EXT_platform_wayland") == m_clientExtensions.end())
+  if (m_clientExtensions.find("_platform_wayland") == m_clientExtensions.end())
```

Comment:
This won't work as you expect.
`find` calls `==` for every item in the set

Reply:
I wasn't paying enough attention and thought it was a string